kernel/common/ezautolinkoperator.php

```diff
@@ -42,15 +54,52 @@ function formatUri( $url, $max )
         }
         return "<a href=\"$url\" title=\"$url\">$text</a>";
     }
+    
+    static function formatUriCallback( $matches )
+    {
+        $method = $matches[1];
+        $www = $matches[2];
+        $domainPath = $matches[3];
+        
+        $url = $method . $www . $domainPath;
+        
+        if ( self::$isLinkWwwUrl && $www == 'www.' )
+        {
+            $url = 'http://' . $www . $domainPath;
+        }
+        
+        $textFull = $url;
```

Comment:
We think this variable name should also be changed. We suggest `$urlLinkText` as a more clear and verbose replacement. We request you change this variable name.